EnvironmentDefinitions/ARMTemplates/App_Service_with_Cosmos/environment.yaml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://github.com/Azure/deployment-environments/releases/download/2022-11-11-preview/manifest.schema.json
+name: App-Service-with-Cosmos
+version: 1.0.0
+summary: Azure APP Service with Cosmos Environment
+description: Deploys an Azure App Service with Cosmos, it is compatible with azd.
+runner: ARM
+templatePath: azuredeploy.json
+
+parameters:
+  - id: environmentName
+    name: environmentName
+    description: 'Name of the Web App.'
+    type: string
+    required: true
+  - id: repoUrl
```

Comment:
I understand this param is specific to the `azd` scenario but we should also make this environment definition user friendly for people not using `azd`.

Perhaps we should add empty string as an allowed value and also introduce a parameter for target language.  This would then prompt the user to select a language that can be reused whether or not they are using from within `azd`.

This same comment applies to all the ARM templates